Python/books.md

```diff
@@ -4,3 +4,4 @@ Learn Python the Hard Way | http://learnpythonthehardway.org/
 Core Python Programming (2nd Edition) | http://www.amazon.com/Core-Python-Programming-Wesley-Chun/dp/0132269937
 How to Think Like a Computer Scientist: Learning with Python | http://www.lulu.com/shop/allen-downey/how-to-think-like-a-computer-scientist-learning-with-python/paperback/product-3487688.html
 Violent Python | http://shop.oreilly.com/product/9781597499576.do
+Directory of free Python ebook | http://pythonbooks.revolunet.com 

```

Comment:
This is invalid URL! Please don't share a common link having lot many books, rather share an original publisher URL of a single book! For example, http://www.headfirstlabs.com/books/hfpython/